### PR TITLE
Make navbar sticky and FlyOut menu full width

### PR DESCRIPTION
**Pull request by florian-lefebvre**
## What kind of change does this PR introduce?

Improvement

## What is the current behavior?

The navbar is not sticky, so the user has to scroll to the top to access the links. Also, the FlyOut component is weird because he is not full width.

## What is the new behavior?

The navbar is now sticky and the FlyOut component is now full width.

## Additional context

I have modified the Nav and the beta page to move the FlyOut menu to the right location, according to the new implementation.
Also, I put a `Fragment` in the Nav to prepare my next PR.

### CODE REVIEW

**Review by MildTomato — CHANGES_REQUESTED** [review]
hey @florian-lefebvre - nice work! just a broken logo needs fixing I think

File: www/components/Nav/index.tsx [hunk 1]
```diff
@@ -183,167 +183,165 @@ const Nav = (props: Props) => {
...
-              >
-                Beta
-              </a>
-              <a
-                href="/pricing"
-                className={`
-                  inline-flex items-center px-1 border-b-2 border-transparent text-sm font-medium
-                  text-gray-500 hover:text-gray-700 hover:border-gray-500 p-5
-                  dark:text-dark-100 dark:hover:border-dark-100
+    <Fragment>
```
[the remark this answers is not in the document]
Comment (florian-lefebvre) [reply www/components/Nav/index.tsx:186]: Oh cool! I come from Vue some I learn React while doing thoses PRs 😄

File: www/components/Nav/index.tsx [hunk 2]
```diff
@@ -183,167 +183,165 @@ const Nav = (props: Props) => {
...
+      {/* Announcement Nav */}
```
[the remark this answers is not in the document]
Comment (florian-lefebvre) [reply www/components/Nav/index.tsx]: @MildTomato I just deleted it

**Comment by MildTomato** [comment]
sorry for delay, i'll take a look

**Review by MildTomato — APPROVED** [review]
lgtm 😄  thanks @florian-lefebvre !

**Pull request merged**